### PR TITLE
[Media] Resolving Issues from Testing

### PR DESCRIPTION
**Pull request by HenriRabalais**
Addresses Issues From:
#5363 - Prevents Users from selecting a site, visit label, or instrument unless a pscid is selected.

This also removes the Site Field from the upload form and edit form in media. Site is redundant since visits are already associated with a site. This removes two unnecessary clicks for the user when uploading a file, and doesn't alter any data going to the database since the site was never saved anyways.

### CODE REVIEW

**Comment by HenriRabalais** [comment]
@maltheism thanks for the testing! I'm blocking this though based on a recent discussion with Rida. We are actually going to remove the 'Site' because it is unnecessary. This seems like the right PR to make that change.

**Comment by HenriRabalais** [comment]
@maltheism unblocked!

File: modules/media/ajax/FileUpload.php [hunk 1]
```diff
@@ -209,7 +208,7 @@ function getUploadFields()
     $user   = \User::singleton();
     $config = \NDB_Config::singleton();
 
-    // Select only candidates that have had visit at user's sites
+    // Select only candidates that have had visit at user's ites
```
Comment (maltheism) [inline modules/media/ajax/FileUpload.php]: ites or sites

Comment (HenriRabalais) [reply modules/media/ajax/FileUpload.php]: oops lol

File: modules/media/ajax/FileUpload.php [hunk 2]
```diff
@@ -108,13 +108,13 @@ function uploadFile()
...
     $visit      = isset($_POST['visitLabel']) ? $_POST['visitLabel'] : null;
     $instrument = isset($_POST['instrument']) ? $_POST['instrument'] : null;
-    $site       = isset($_POST['forSite']) ? $_POST['forSite'] : null;
     $dateTaken  = isset($_POST['dateTaken']) ? $_POST['dateTaken'] : null;
     $comments   = isset($_POST['comments']) ? $_POST['comments'] : null;
     $language   = isset($_POST['language']) ? $_POST['language'] : null;
 
     // If required fields are not set, show an error
-    if (!isset($_FILES) || !isset($pscid) || !isset($visit) || !isset($site)) {
+    if (!isset($_FILES) || !isset($pscid) || !isset($visit) ||
```
Comment (johnsaigle) [inline modules/media/ajax/FileUpload.php]: Just FYI you can pass multiple argument to isset

```php
if (!isset($arg1, $arg2, ...))
```

Comment (HenriRabalais) [reply modules/media/ajax/FileUpload.php]: love it

Comment (HenriRabalais) [reply modules/media/ajax/FileUpload.php]: @johnsaigle looks like that should fix the phpcs errors as well

**Comment by HenriRabalais** [comment]
@johnsaigle resolved issue

**Comment by ridz1208** [comment]
blocked by discussion on individual issues

**Comment by ridz1208** [comment]
@HenriRabalais 

could you add more details on how you solved this : 

> #5363 - Prevents Users from selecting a site, visit label, or instrument unless a pscid is selected.

Cause from what I can tell simply removing the site selection should be sufficient to solve the error reported by Leigh

**Comment by driusan** [comment]
Scratch what I said about the site being for uploading content for other sites.. IBIS uses a different module for reliability videos, so inferring the site in this one should be fine. (The other points stand.)

**Comment by HenriRabalais** [comment]
@ridz1208 if you scroll down and read the comments on the issues (#5363) I mention that I discovered that the same issue that involves sites also occurs for both visit label and instruments. Should I update the issue or make a new one to reflect those similar problems?

**Comment by HenriRabalais** [comment]
@driusan @ridz1208 I have removed language and instrument as required fields from the form. Language as a required field can be addressed in another PR if necessary.

**Pull request merged**